ui/components/multichain/global-menu/global-menu.js

```diff
@@ -253,7 +253,6 @@ export const GlobalMenu = ({ closeMenu, anchorElement, isOpen }) => {
       >
         {t('allPermissions')}
       </MenuItem>
-
```

Comment:
Not needed, can remove to make PR smaller.

Reply:
Added back https://github.com/MetaMask/metamask-extension/pull/26635/commits/9b8a4843e14e13f22607be4ca864747d524a1948